### PR TITLE
[DEX] update

### PR DESCRIPTION
**Pull request by deadlypants1973**
### Summary

update for launch

### Screenshots (optional)

Screenshot 2024-12-19 at 15 30 56

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

**Review by angelampcosta — APPROVED** [review]

File: src/content/docs/cloudflare-one/account-limits.mdx [hunk 1]
```diff
@@ -66,7 +66,7 @@ This page lists the default account limits for rules, applications, fields, and
 
 | Feature           | Limit |
 | ----------------- | ----- |
-| Tests per account | 10    |
+| Tests per account | Free Plan: 10, <br/>Pro & Business Plans: 30, <br/>Enterprise: 50   |
```
Comment (renahlee) [inline src/content/docs/cloudflare-one/account-limits.mdx]: Hi there! We've had some hold up on the entitlements changes and after speaking to my PM (Guy), we plan to descope this from DEX GA. Happy to approve removal of the `beta` badge once this is removed :)

Comment (ranbel) [inline src/content/docs/cloudflare-one/account-limits.mdx]:
```suggestion
| Tests per account | 10   |
```

**Review by renahlee — APPROVED** [review]

File: src/content/docs/cloudflare-one/account-limits.mdx (hunk 1 above)
Comment (renahlee) [reply src/content/docs/cloudflare-one/account-limits.mdx]: Hi @ranbel!

Limits were finally updated - would it be possible to re-add this change?

Comment (ranbel) [reply src/content/docs/cloudflare-one/account-limits.mdx]: Yes! I created PCX-15936 for Kate and added you as the reporter.

**Pull request merged**